src/solver/summary.h

```diff
@@ -53,6 +54,11 @@ class summaryt
   predicatet bw_transformer; // backward summary (over- or under-approx)
   predicatet bw_invariant; // backward invariant (over- or under-approx)
 
+  std::unique_ptr<domaint> fw_domain_ptr;
+  std::unique_ptr<domaint::valuet> fw_value_ptr;
```

Comment:
Because the domain stores the variables and the the value stores the abstract values for the variables. We need both for instrumentation of GOTO programs with invariants.